doc/nrf/libraries/dfu/dfu_multi_image.rst

```diff
@@ -0,0 +1,33 @@
+.. _lib_dfu_multi_image:
+
+DFU Multi Image
+###############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The DFU Multi Image library provides an API for writing a DFU Multi Image package that is being downloaded in chunks of an arbitrary size.
+The DFU Multi Image package is a simple archive file consisting of a CBOR-based header describing contents of the package, followed by a number of updates images, such as firmware images for different MCU cores.
+Images included in a DFU Multi Image package are identified by user-assigned numeric identifiers.
+The library provides a means for a user to register custom functions for writing package data that belongs to an image with a given identifier.
+That makes the library a general-purpose solution as it makes no assumptions with respect to formats of images included in a written package.
+
+Configuration
+*************
+
+To enable the DFU Multi Image library, set the :kconfig:option:`CONFIG_DFU_MULTI_IMAGE` Kconfig option.
+
+To configure the maximum number of images that the DFU Multi Image library is able to process, use the :kconfig:option:`CONFIG_DFU_MULTI_IMAGE_MAX_IMAGE_COUNT` Kconfig option.
+
+To enable building the DFU Multi Image package that contains common update images, such as the application core firmware, the network core firmware, or MCUboot images, set the :kconfig:option:`CONFIG_DFU_MULTI_IMAGE_PACKAGE_BUILD` Kconfig option.
+
```

Comment:
Dependencies are required by the library template, so maybe let's add this section here with a link to zcbor?
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/library_template_README.html#dependencies

Reply:
Ok, added.